mantidproject/mantid
# Fixes Q, Wavelength settings not scaling

**Description of work.**
Removes a hidden layout object that was incorrect, which allowed Qt
designer to correctly layout the page. This will now scale on high-res monitors.

Note: I'm intentionally targetting 6.3 despite it being found during 6.2 testing for a few reasons:
- The diff is large because of Qt having to shift elements up 1 layer of the tree
- I'm worried about any lost tooltips / names from such a large diff
- It's only a few lucky developers who have 4k monitors. Most of the SANS group is using RDP to IDaaS currently.

Found in #32275

**To test:**
- Open the ISIS SANS GUI
- Go to settings
- Check Q, Wavelength scales on a high-resolution monitor

This does not require release notes because it's not been noticed outside the development team and 6.3 release notes don't exist yet.



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
